src/program/snabbvmx/lwaftr/lwaftr.lua

```diff
@@ -141,11 +141,18 @@ function run(args)
 
    local vlan = conf.settings and conf.settings.vlan or false
 
+   local mtu = DEFAULT_MTU
+   if lwconf.ipv6_mtu then
+     mtu = lwconf.ipv6_mtu + 14
```

Comment:
Better to use constant `ethernet_header_size` in `apps/lwaftr/constants.lua`. Also if vlan is true, should add increase mtu by 4.